libmscore/chord.cpp

```diff
@@ -131,6 +131,18 @@ int Chord::downLine() const
       }
 
 //---------------------------------------------------------
+//   bottomLine
+//    the line position corresponding to the bottom line of the staff
+//---------------------------------------------------------
+
+int Chord::bottomLine() const
```

Comment:
This doesn't belong here. It's better to compute it when we need it since we need it once for now. Also it's not the bottomLine of the chord but the one of the staff. It would be better to have a function to give back the bottom line of a staff and do
```
Staff* staff = score()->staff(vStaffIdx());
int bottomLine = staff->bottomLine();
```